### PR TITLE
Add Get current Url in browser

### PR DESCRIPTION
**Pull request by pasalino**
When you use the library in browser, you must get url from window if you would use the library for current url. 

This change add feature to create object directly with current page url.

The URLObject if called without parameters, get current url from browser. 
If you are in server side, the object throw error becouse process.browser is false.

### CODE REVIEW

**Comment by stecb** [comment]
Hey @pasalino! Awesome feature, thanks! I’m from mobile as on holiday, will check it after the 1st Sept. If in the meantime @ksnll wants to have a look and think it’s all good we can deffo merge it.

Thanks again! 🙌

**Comment by stecb** [comment]
(Side note: worth having an eslintrc + husky lint check before commint)

**Comment by stecb** [comment]
@pasalino definitely! Gonna add everything once I’m back :)

File: package-lock.json [hunk 1]
```diff
@@ -47,18 +129,24 @@
...
       "integrity": "sha1-D2RYS6ZC8PKs4oBiefTwbKI61zw=",
       "dev": true,
       "requires": {
-        "assertion-error": "^1.0.1",
-        "check-error": "^1.0.1",
-        "deep-eql": "^3.0.0",
-        "get-func-name": "^2.0.0",
-        "pathval": "^1.0.0",
-        "type-detect": "^4.0.0"
+        "assertion-error": "1.1.0",
```
[the remark this answers is not in the document]
Comment (pasalino) [reply package-lock.json]: I don't knwon nothing about this topic. I'm agnostic about it.

**Comment by pasalino** [comment]
All change request is done!

**Pull request merged**